daemon/api_aspects.go

```diff
@@ -53,9 +54,28 @@ func getAspect(c *Command, r *http.Request, _ *auth.UserState) Response {
 	st.Lock()
 	defer st.Unlock()
 
+	_, err := getDatabag(st, account, bundleName)
+	if err != nil {
+		if errors.Is(err, state.ErrNoState) {
+			return toAPIError(&aspects.AspectNotFoundError{Account: account, BundleName: bundleName, Aspect: aspect})
+		}
+		return toAPIError(err)
+	}
+
+	schema := aspects.NewJSONSchema()
+	getter := bagGetter(st, account, bundleName)
+	setter := func(aspects.JSONDataBag) error {
+		return nil
+	}
+
+	tx, err := aspects.NewTransaction(getter, setter, schema)
```

Comment:
this code probably belong to a helper in aspsectstate for consistency of responsibility levels